seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -1178,8 +1178,8 @@
       "description": [
         "Your cat photo currently has sharp corners. We can round out those corners with a CSS property called <code>border-radius</code>.",
         "<hr>",
-        "You can specify a <code>border-radius</code> with pixels. Give your cat photo a <code>border-radius</code> of <code>10px</code>.",
-        "Note: this waypoint allows for multiple possible solutions. For example, you may add <code>border-radius</code> to either the <code>.thick-green-border</code> class or <code>.smaller-image</code> class."
+        "One of the units you can use to specify the radius of a border is with pixels. Give your cat photo a <code>border-radius</code> of <code>10px</code>.",
```

Comment:
Yes - you can use other measurements in addition to pixels. But they are not relevant for this challenge.